backend_new/src/dtos/multiselect-questions/multiselect-question-filter-params.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseFilter } from '../shared/base-filter.dto';
+import { Expose } from 'class-transformer';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+import { MultiselectQuestionFilterKeys } from '../../enums/multiselect-questions/filter-key-enum';
+import { MultiselectQuestionsApplicationSectionEnum } from '@prisma/client';
+
+export class MultiselectQuestionFilterParams extends BaseFilter {
+  @Expose()
+  @ApiProperty({
+    type: String,
+    example: 'uuid',
+    required: false,
+  })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  [MultiselectQuestionFilterKeys.jurisdiction]?: string;
+
+  @Expose()
+  @ApiProperty({
+    type: String,
```

Comment:
Can this be more strongly typed?

Reply:
so this is expecting a jurisdiction's ID which is a UUID/string, I couldn't find documentation on making this a type like UUID so I had to keep it typed as a string 

I did add the ISUUID() check though. 

lemme know if that works for ya or if you want me to look into maybe making a custom type or something similar